src/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -27,20 +25,12 @@ public WebHeaderCollection Headers
             }
         }
 
+        public Encoding ContentEncoding { get; set; }
```

Comment:
Odd... I inverted the Unix/Windows checks and did find-all-references for this, but got no results. Reloaded the solution and got this result. Thanks for spotting this - it's not currently tested as we don't have any tests for when we try to connect to HttpListener but fail.